Updating old permissions (#61)
Basis-Theory/basistheory-integration-guides: collections/guides/setup_first_reactor/setup_first_reactor.md

<span class="base-alert warning">
  <span>

>     To run a Reactor, an application needs `token:<classification>:read:<impact_level>` and `token:<classification>:use:reactor` permissions. For this example you will need `token:pci:read:<classification>` and `token:pci:use:reactor`.

Reviewer: ```suggestion
    To run a Reactor, an application needs `token:<classification>:use:reactor` permission. For this example the application will need `token:pci:use:reactor` permission.
```
